### DISCUSSION TITLE
Welcome to OpenROAD-flow-scripts Discussions!

### DISCUSSION THREAD

**Discussion by maliberty** [The-OpenROAD-Project/OpenROAD-flow-scripts#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Raynchowkw** [comment]
Hi dear community!
I am a graduate student doing a project to graduate soon.
I am investigating detailed router and trying to using the fancy tritonroute as infrastructure to dump out access point information.
I am still learning C++ and docker. Hope to learn from all of you.

**Reply by Raynchowkw** [reply]
If there are folks who are also trying to dump out access points information for each cell in TritonRoute, let me know and we can learn together.
